set.mm

```diff
@@ -254354,6 +254365,15 @@ irreducible elements (see ~ df-irred ).  (Contributed by Mario Carneiro,
   isfld $p |- ( R e. Field <-> ( R e. DivRing /\ R e. CRing ) ) $=
     ( cdr ccrg cfield df-field elin2 ) ABCDEF $.
 
+  ${
+    fldcrngd.1 $e |- ( ph -> R e. Field ) $.
+    $( A field is a commutative ring.  EDITORIAL:  Shortens ~ recrng .  Also
```

Comment:
The "EDITORIAL" should be solved when a theorem is moved to main.

Reply:
Ok, I did the EDITORIAL part, but I don't understand "~ fldcrng is misnamed". That name sounds right to me. What would you rename it with?

Reply:
probably fldcrngo, since it uses Fld and CRingOps instead of Field and CRing

Reply:
Ok, renamed `~ fldcrng` to `~ fldcrngo` !